cypress/integration/example.spec.ts

```diff
@@ -0,0 +1,9 @@
+/// <reference types="cypress" />
```

Comment:
This reference will be used in all testing files, thus leading to code repetition. Cypress types could be added to `.tsconfig`, making them global

Reply:
I tried, but adding anything to the `types` field in `tsconfig` results in breaking the types for unit tests. Check the comment above

Reply:
Anyway, we will be adding E2E tests into the Cypress folder, so what about a new `.tsconfig` in the `cypress` directory. It can extend the base config and add Cypress types only for this directory

Reply:
Right, I forgot about that possibility. Done